New groundtruth

FloChiff opened the pull request:
This is an improved version of the ground truth from the DAHN corpus, with more precise file names and images of better quality that are also available online.

FloChiff committed:
Added a better version of the ground truth

FloChiff committed:
Added informations about the images

FloChiff committed:
Renamed the folders

FloChiff committed:
Changed the tree structure

FloChiff committed:
Updated the READMEs

FloChiff commented:
The last three commits modified the tree structure so that it is more clear.

FloChiff committed:
Update htr-united.yml

alix-tz commented:
This is great! Do you think you can add a column in the description of the dataset to indicate whether the batch includes images or not? (or when they are available they are only in Nakala, to indicate when they are available and when they are not)

FloChiff commented:
I put a "Images" section under the table presenting the dataset where I explain that all the images are available in NAKALA and every batch folder have the links to the images in NAKALA, as said in the description also. This is also a way to not put the images twice as there is "PAGE" and "ALTO" in each folder. Is that not enough ?

FloChiff committed:
Added the images

alix-tz committed:
Update htr-united.yml

- parsing ALTO files rather that PAGE files since that's the main format you use
- adding character table generation

alix-tz committed:
deactivated segmonto validation by HTRVX

FloChiff committed:
Merge pull request #7 from HTR-United/alix-tz-patch-1

Update workflow specs
